recipe/meta.yaml

```diff
@@ -264,7 +268,6 @@ outputs:
         # Workaround to prevent conflicts seen in
         #  https://github.com/conda-forge/gdal-feedstock/pull/543
         - openssl
-        - swig
```

Comment:
Actually from the build error on https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=703871&view=results, it looks like swig must perhaps be kept in the host section, in addition to being in the build one ?

Reply:
trying